amity/amity.py

```diff
@@ -0,0 +1,255 @@
+import os, sys, inspect
+from sqlalchemy.orm import relationship, backref
+currentdir = os.path.dirname(os.path.abspath(inspect. \
+    getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir) 
+from rooms import Room, LivingSpace, Office 
+from persons import Person, Fellow, Staff
+import random
```

Comment:
**Code Quality - Nitpick:** Adhere to the [PEP8 import recommendations](https://www.python.org/dev/peps/pep-0008/#imports).